### ISSUE TITLE
Need to put caching infront of all /api/ calls by default

### ISSUE THREAD

**Issue by Markavian** [connected-web/product-monitor#12]
Because its too deadly to let some of these heavy weight calls fan out to the wider web, especially when multiple clients are open in several browser windows.

Also, it may be worth caching some API responses to files to make the server more resilient to restarts.

**Comment by Markavian** [comment]
I put this filter in as a test, but it creates an invalid path record on the /docs/api endpoint. 

``` js
  // Check cache before every API call
  server.all('/api/*', apiController.checkCache);
```

``` js
// Check the cache before each and every API call
instance.checkCache = function (req, res, next) {
  console.log("Preparing to cache" + req.originalUrl);
  next();
}
```

I might keep the `/api/*` record and return a JSON response listing the APIs known directly to the `apiController` - caching can still be applied through this filter.

**Comment by Markavian** [comment]
Fixed in 1.7.1 using `apicache`
- Added `apicache` as middleware to all registered API endpoints
- Default caching is set to `1 hour` for all registered `/api/` routes
- Added optional `instance.cacheDuration` property that can be set cacheDuration for individual endpoints.
- Fixed broken documentation link in `index.content.html` when starting a new project.
